docs/src/solvers/sde_solve.md

```diff
@@ -147,6 +153,18 @@ For `SRA` and `SRI`, the following option is allowed:
 
 * `tableau`: The tableau for an `:SRA` or `:SRI` algorithm. Defaults to SRIW1 or SRA1.
 
+#### S-ROCK Methods
+
+- `SROCK1` - is a fixed step size stabilized explicit method for stiff problems. Defaults to
+  solving th Ito problem but `SROCK1(interpretation=:Stratonovich)` can make it solve
+  the Stratonovich problem. Strong order of convergence is 0.5 and weak order 1, but is
+  optimised to get order 1 in case os scalar/diagonal noise.
+- `SROCKEM` - is fixed step Euler-Mayurama with first order ROCK stabilization thus can
+  handle stiff problems. Only for Ito problems. Defaults to strong and weak order 1.0,
+  but can solve with weak order 0.5 as `SROCKEM(strong_order_1=false)`.
+- `SROCK2` - is a weak second order and strong first order fixed step stabilized method for
```

Comment:
isn't there more?

Reply:
But they are not fully functional.

Reply:
Add a note about that in the description. Then we can delete it when they are done. It's better to just document all we can while we're thinking about it.

Reply:
Added!